debian/README.Debian

```diff
@@ -5,7 +5,26 @@ This is a binary-only package produced by the wasm waterfall which
 is hosted at https://wasm-stat.us/.  The code that runs the build
 is hosted at https://github.com/WebAssembly/waterfall.
 
+This package includes the following components:
+
+ - upstream llvm including wasm backend (/opt/wasm/llvm)
+ - fastcmp llvm (/opt/wasm/fastcomp)
+ - emscripten (/opt/wasm/emscripten)
+ - binaryian (/opt/wasm/bin/)
+ - wabt (/opt/wasm/bin/)
+ - emscripten luancher script (/opt/wasm/bin/emcc)
+
 The versions of each of the source package that were used to produce
-this package are documented in /usr/share/wasm/buildinfo.json.
+this package are documented in /opt/wasm/buildinfo.json.
+
+As well as installing files under /opt/wasm the package also uses
+the debian alternatives mechanism to install /usr/bin/emcc and
+/usr/bin/em++ which point to the scripts in /opt/wasm/bin/emcc.
+
+These wrapper scripts allow emcc and em++ to be used without any
+EM_CONFIG or ~/.emscripten_config (i.e. they provide a pre-configured
+version of the toolchain).  The preconfigured toolchain will use
+upstream LLVM when EMCC_WASM_BACKEND is set and fastcomp otherwise,
+allowing the toolchain to produce both wasm and asmjs.
```

Comment:
I forget what we settled on: will it respect the `--em-config` flag or `EMSCRIPTEN_CONFIG` for the user to override the config? (probably should document that either way).